website/src/components/structure/main-content-container/main-content-container.tsx

```diff
@@ -11,10 +11,20 @@ export const MainContentContainer: FunctionComponent = ({ children }) => {
   const dispatch = useDispatch();
 
   useEffect(() => {
+    dispatch(
+      hasScrolled({
+        yScrollPosition: 0,
+      })
+    );
+
     const handleScroll = () => {
+      if (!ref.current || ref.current.scrollTop === undefined) {
```

Comment:
Just `ref.current?.scrollTop === undefined` would be enough 👍

Reply:
No, because `ref.current` is initialized to `null` and this check would only catch `undefined` then (`===`).

Reply:
true